docs/source/markdown/options/network.md

```diff
@@ -74,3 +74,6 @@ Valid _mode_ values are:
     - **pasta:-T,5201**: enable forwarding of TCP port 5201 from container to
         host, using the loopback interface instead of the tap interface for improved
         performance
+
+    NOTE: For backwards compatibility reasons if you have a network named `pasta` podman will use this instead.
```

Comment:
```suggestion
    NOTE: For backward compatibility reasons, if you have a network named `pasta,` Podman will use this instead of the pasta binary.
```

Reply:
I guess the choice between _backward_ and _backwards_ should be free here, given that in AmE _backwards_ is more commonly used for the adjective, and that _backward_, conversely, occurs more frequently as adjective in BrE. The rest of the paragraph doesn't happen to use words with a preferred or specific spelling in either one.

Also, if changed:
- `` `pasta`, ``, not `` `pasta,`  ``
- I'd rather call it _mode_ rather than _binary_, given that we call those network _modes_ just above

Just for clarity, I'm not suggesting this should be changed at all -- to me it's clear and terse as it is.

Reply:
Ooops, and thanks for the comma quote vs quote comma catch.  I've edited my comment.  My $.02, I think the note as it stands is way too terse, especially for someone like me who has practically no idea what pasta is.  I'm fine with mode over binary if that fits better.

Reply:
fixed